src/trustyai/utils/tyrus.py

```diff
@@ -131,7 +140,17 @@ def __init__(
         if self.notebook:
             output_notebook()
         else:
-            output_file(filename="trustyai_dashboard.html", title="TrustyAI Dashboard")
+            if kwargs.get("filepath") is None:
+                self.tmp_dir = (
+                    tempfile.TemporaryDirectory()  # pylint: disable=consider-using-with
+                )
+                self.filepath = self.tmp_dir.name
+            else:
+                self.filepath = kwargs["filepath"]
+            output_file(
+                filename=self.filepath + "/trustyai_dashboard.html",
```

Comment:
I think it would be best to use something like `os.path.join` rather than adding the strings directly.

Reply:
yep, will do